internal/pb/fs.go

```diff
@@ -9,7 +9,12 @@ import (
 	"path/filepath"
 )
 
-func tarTypeFromMode(mode fs.FileMode) byte {
+const (
+	TarDeleted = 'D'
+	TarCached  = 'C'
```

Comment:
Sounds like we're good to use `'C'`

<img width="638" alt="image" src="https://user-images.githubusercontent.com/515110/194598192-f9a07072-9338-48d0-9497-f943e4f420c3.png">